examples relying on pygmsh broken with Gmsh 4 now using format MSH4

The two examples, ex12 and ex13, using pygmsh.generate_mesh are broken.  In each case the call raises 
> AssertionError: Need mesh format 2

from `meshio.gmsh_io._read_header`.

I think this might be the result of upgrading to Gmsh 4.  The MSH files are binary but the first two lines are
```
$MeshFormat
4 1 8
```

The 4 is what is asserted to be 2 by meshio, so I think this is a [https://github.com/nschloe/meshio](meshio) issue.  I'll report and try to fix it there.

--- a/examples/ex13.py
+++ b/examples/ex13.py
@@ -46,7 +46,7 @@ geom.add_physical_surface(
     geom.add_plane_surface(geom.add_line_loop(lines)), 'domain')
 
 pts, cells, _, cell_data, field_data = generate_mesh(
-    geom, prune_vertices=False)
+    geom, prune_vertices=False, extra_gmsh_arguments=['-format', 'msh2'])
 
 mesh = MeshTri(pts[:, :2].T, cells['triangle'].T)
 boundaries = {bc:

